lib/web_ui/lib/src/engine/canvaskit/embedded_views.dart

```diff
@@ -246,11 +246,26 @@ class HtmlViewEmbedder {
     return head;
   }
 
-  void _applyMutators(MutatorsStack mutators, html.Element embeddedView) {
+  /// Clean up the old SVG clip definitions, as this platform view is about to
+  /// be recomposited.
+  void _cleanUpClipDefs(int viewId) {
+    if (_svgClipDefs.containsKey(viewId)) {
+      final html.Element clipDefs =
+          _svgPathDefs!.querySelector('#sk_path_defs')!;
+      for (String id in _svgClipDefs[viewId]!) {
+        clipDefs.querySelector('#$id')?.remove();
```

Comment:
Not sure if this is going to be more performant, but something that maybe doable is:

```dart
clipDefs.querySelectorAll('#id1, #id2, #id3'); // returns all the IDs in a single querySelectorAll call
```

Or maybe filtering the `children` property of the `clipDefs`, since we know that the clip defs are direct descendants of the clipDef tag:

```dart
clipDefs.children = clipDefs.children.where((child) => ! _svgClipDefs[viewId]!.contains(child.id)).toList();
```

_(Assuming `_svgClipDefs` is a `Map<int, Set<String>>` instead of a map of int to List)_

Anyway, the comment above might be an un-needed micro-optimization, since it's most likely that this'll only call a few selectors per frame, and not a huge list, so LGTM!

Reply:
Done.